src/utils/download_output.ts

```diff
@@ -0,0 +1,77 @@
+import { GQL_URL, gatewayUrl } from '../constants';
```

Comment:
We can have a tech_debt to implement retries in these requests

Reply:
I don't think it worth it. It's just a couple calls that you can retry running the script again.

Reply:
Exactly! It's not worth it at the moment, we can have that as a tech_debt ticket